src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx

```diff
@@ -26,7 +29,13 @@ export default function OpeningPeriod({
           </div>
         </div>
         <div className="opening-period-title opening-period-row-column">
-          <h4>{datePeriod.name?.fi}</h4>
+          {name ? (
+            <h4>{name}</h4>
+          ) : (
+            <h4 className="text-danger">
+              {getMissingText({ language, label: 'nimi' })}
```

Comment:
I wonder if _displayMissingText_ or _displayLangVersionNotFound_ could be a more descriptive name for the function `getMissingText`? To me _getMissingText_ gives maybe some false hope, that we could be able to still get the text that is missing? 😅

Reply:
Yes, I struggled with the naming as well. Maybe _displayLangVersionNotFound_?

Reply:
displayLangVersionNotFound sounds good

Reply:
Fixed